section2/exercises/if-statements.js

```diff
@@ -82,14 +100,24 @@ When cupsOfFlour = 2 and hasSauce = true, your program should log "I can make pi
 When cupsOfFlour = 3 and hasSauce = true, your program should log "I can make pizza";
 */
 
-var cupsOfFlour = 1;
+// var cupsOfFlour = 1;
 var hasSauce = true;
-
+// var cupsOfFlour = 2;
+var cupsOfFlour = 3;
+// var hasSauce = false;
 // Write your conditional here
-
+if (cupsOfFlour === 1 && hasSauce === true) {
```

Comment:
Similar to my push for you on the previous solution - could you rewrite so there is no repeated code??